dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -108,12 +110,18 @@ public int createPullReq(GHRepository origRepo, String branch,
             log.warn("Handling error with pull request creation... {}", e.getMessage());
             JsonElement root = JsonParser.parseString(e.getMessage());
             JsonArray errorJson = root.getAsJsonObject().get("errors").getAsJsonArray();
-            String error = errorJson.get(0).getAsJsonObject().get("message").getAsString();
+            String error = "";
+            if (errorJson.get(0).getAsJsonObject().has("message")) {
+                error = errorJson.get(0).getAsJsonObject().get("message").getAsString();
+            } else {
+                // This case usually happens when the PR head has invalid data. Deleting the forked repo resolves it.
+                error = PR_INVALID_CODE;
+            }
             log.info("error: {}", error);
             if (error.startsWith("A pull request already exists")) {
                 log.info("NOTE: {} New commits may have been added to the pull request.", error);
                 return 0;
-            } else if (error.startsWith("No commits between")) {
+            } else if (error.startsWith("No commits between") || error.startsWith(PR_INVALID_CODE)) {
```

Comment:
Create a list of error prefixes and use stream operations

```
List<String> prefixes = Arrays.asList(PR_INVALID_CODE, "No commits between")

if(prefixes.anyMatch(prefix -> errors.startsWith(prefix))
```
This way if you add more prefixes the if statement doesn't change.